test/com/google/javascript/jscomp/parsing/ParserTest.java

```diff
@@ -3657,6 +3657,14 @@ public void testExport() {
     parseError("export * as s from './someModule';", "'from' expected");
   }
 
+  public void testExportTypescriptKeyword() {
```

Comment:
Can you add a test for: `"import { namespace } from './input0.js';"`? Thanks @lauraharker for catching that.

Reply:
Ok, done. Lemme know if you need anything else or there are style/doc concerns.